app/models/shipit/merge_request.rb

```diff
@@ -112,17 +112,17 @@ def self.schedule_merges
     def self.extract_number(stack, number_or_url)
       case number_or_url
       when /\A#?(\d+)\z/
-        $1.to_i
+        Regexp.last_match(1).to_i
```

Comment:
I just disabled that silly cop on master. Please revert that change.

Reply:
[There was a another definition of that cop later on in the `.rubocop.yml`](https://github.com/Shopify/shipit-engine/pull/1095/commits/5dd9f8a1d03f1d2ec532c83ed3383dd1fa451789#diff-11a0d7906801d9dea0eccb85667ad811) which overrode the changes in https://github.com/Shopify/shipit-engine/commit/e0acb4e68805d04b119001a38c765c4c3358461b. I've removed the later cop definition so the `Style/PerlBackrefs` cop is totally disabled for the project as part of this body of work. Hope that doesn't muddy the waters too much.

Reply:
Thanks.